vcd/resource_vcd_vm_placement_policy.go

```diff
@@ -439,3 +440,37 @@ func setVmPlacementPolicy(_ context.Context, d *schema.ResourceData, vcdClient *
 	log.Printf("[TRACE] VM Placement Policy read completed: %s", policy.Name)
 	return nil
 }
+
+// getVgpuFilterToPrepend gets a vGPU Policy filter set to `isVgpu` if API version of target VCD is greater than 36.2 (VCD 10.3.2).
```

Comment:
Shouldn't all these functions be in go-vcloud-director?
(getVgpuFilterToPrepend, getVgpuFilter, getFriendlyErrorIfVmPlacementPolicyAlreadyExists)

Reply:
I've moved `getFriendlyErrorIfVmPlacementPolicyAlreadyExists` to go-vcloud-director.